### PR TITLE
add variants grub_file in SLES and Ubuntu cfg,delete variants grub_file in RHEL7.0 cfg

### PR DESCRIPTION
**Pull request by PyLearner**
add variants grub_file in SLES and Ubuntu cfg,delete variants grub_file in RHEL7.0 cfg

### CODE REVIEW

**Comment by ypu** [comment]
Hi @PyLearner how about put the grub_file to SLES.cfg and ubuntu.cfg, seems they are all the same for them.

File: shared/cfg/guest-os/Linux/RHEL/7.0/x86_64.cfg [hunk 1]
```diff
@@ -1,5 +1,4 @@
 - x86_64:
-    grub_file = /boot/grub/grub.conf
```
Comment (CongSmile) [inline shared/cfg/guest-os/Linux/RHEL/7.0/x86_64.cfg]: Hi meyang,

Why remove this line here?

**Comment by CongSmile** [comment]
Hi meyang,

Your pullreq is conflicted with #2064 .
Could you update yours?

Thanks.

**Comment by CongSmile** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Pull request closed without merging**